src/test/resources/net/sf/jabref/importer/fileformat/MedlinePlainImporterTestDOI.bib

```diff
@@ -1,3 +1,3 @@
 @TechReport{,
-  DOI                      = {10.1016/j.cpr.2005.02.002}
+  article-id                      = {doi:10.1016/j.cpr.2005.02.002}
```

Comment:
Sorry, but I don't really get what you're trying to say :smile: .
Check out `DOI.java` it includes the regex for a valid DOI. Typically all our DOIs are just the DOI starting with the vendor code, i.e., 10.1016/j.cpr.2005.02.002.
From my understanding the removed line is the right result, except that the key should be lowercase but correct me if I'm wrong and don't understand the use case here!